### DISCUSSION TITLE
MySQL (MariaDB) Error when starting the logon server

### DISCUSSION THREAD

**Discussion by Phthisicus** [azerothcore/azerothcore-wotlk#16889] [Guides & Tips]
Hi Everyone,

Can someone explain this error. I have tried this with MariaDB 10.6 and upgraded tot 10.8 but with no luck.

Nowhere there is indicated that is use Server Version 50505. I used the automated installation script with a few changes for locations where he needs to put the files.

`Found server version: 50505. Server compiled with: 80033.`

At this point i am a little stuck.

![Error](https://github.com/azerothcore/azerothcore-wotlk/assets/76635863/ac2d45ca-96a8-4e51-b0fd-908cd3f0d5e2)

**Comment by Winfidonarleyan** [comment]
No need to install multiple libraries on the same machine

**Comment by Phthisicus** [comment]
What do you mean by that?

As far as my understanding so far searching, is that is had someting to do with a prefix 5.5.5 thats been added with mariadb. The authserver therefor thinks its version 5.5 instead of 10.6.

https://github.com/php/php-src/issues/7972
